assessment.js

```diff
@@ -1,8 +1,13 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var answer = '';
 
-while (answer !== '' && answer !== 'SeCrEt') {
+while (answer !== '6' && answer !== 'SeCrEt') {
   answer = ask("Guess my secret? ");
+  if (answer === '6') } {
+    console.log ("that's the number")
+  }
 }
+
+(didn't finish)
```

Comment:
Gotcha. Let me know if you want to tackle this again - happy to give you feedback!